### PR TITLE
solve the unconsistent comments 

### PR DESCRIPTION
**Pull request by yuanjumao**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: src/main/org/apache/tools/ant/ComponentHelper.java [hunk 1]
```diff
@@ -452,12 +452,7 @@ public void checkTaskClass(final Class<?> taskClass) throws BuildException {
...
 
     /**
-     * Adds a new datatype definition.
-     * Attempting to override an existing definition with an
-     * equivalent one (i.e. with the same classname) results in
-     * a verbose log message. Attempting to override an existing definition
-     * with a different one results in a warning log message, but the
-     * definition is changed.
+     * Update the component definition table with a new or modified definition.
      *
```
Comment (bodewig) [inline src/main/org/apache/tools/ant/ComponentHelper.java]: This replaces a somewhat lengthy detailed description of what happens by a less clear one-liner. Why is this better?

File: src/main/org/apache/tools/ant/UnknownElement.java [hunk 2]
```diff
@@ -151,11 +151,7 @@ public RuntimeConfigurable getWrapper() {
     }
 
     /**
-     * Creates the real object instance and child elements, then configures
-     * the attributes and text of the real object. This unknown element
-     * is then replaced with the real object in the containing target's list
-     * of children.
-     *
+     * Configure the given object from this UnknownElement
```
Comment (bodewig) [inline src/main/org/apache/tools/ant/UnknownElement.java]: This replaces a somewhat lengthy detailed description of what happens by a less clear one-liner. Why is this better?

File: src/main/org/apache/tools/ant/types/XMLCatalog.java [hunk 3]
```diff
@@ -197,10 +197,7 @@ public Path createClasspath() {
     }
 
     /**
-     * Allows simple classpath string.  Not allowed if this catalog is
-     * itself a reference to another catalog -- that is, a catalog
-     * cannot both refer to another <em>and</em> contain elements or
-     * other attributes.
+     * Append the contents of the other Path instance to this.
```
Comment (bodewig) [inline src/main/org/apache/tools/ant/types/XMLCatalog.java]: This replaces a somewhat lengthy detailed description of what happens by a less clear one-liner. Why is this better?

File: src/main/org/apache/tools/ant/types/selectors/modifiedselector/ModifiedSelector.java [hunk 4]
```diff
@@ -686,9 +686,7 @@ public void addParam(Parameter parameter) {
 
 
     /**
-     * Defined in org.apache.tools.ant.types.Parameterizable.
-     * Overwrite implementation in superclass because only special
-     * parameters are valid.
+     * Set all the Parameters for this custom selector, collected by\n the ExtendSelector class.
```
Comment (bodewig) [inline src/main/org/apache/tools/ant/types/selectors/modifiedselector/ModifiedSelector.java]: there seems to be something wrong with the new comment (literal `\n`) and I don't understand what ExtendSelector refers to.

**Pull request closed without merging**